decidim-admin/app/views/decidim/admin/features/_feature.html.erb

```diff
@@ -9,37 +9,25 @@
   </td>
   <td class="table-list__actions">
     <% if feature.manifest.admin_engine %>
-      <%= link_to manage_feature_path(participatory_process, feature), data: { tooltip: true, disable_hover: false }, title: t("actions.manage", scope: "decidim.admin"), class: "action-icon action-icon--manage" do %>
-        <%= icon "pencil" %>
-      <% end %>
+      <%= icon_link_to "pencil", manage_feature_path(participatory_process, feature), t("actions.manage", scope: "decidim.admin"), "action-icon--manage" %>
```

Comment:
Oh, nice xD. Anyway, I think specifying the whole class might be a bit redundant. Is there any case where you need to override the whole class name? If that's the case, maybe you could use a class: "whatever" option instead of this? Not a big fan of long lists of arguments when some of them are/could be optional.